[19.9] Add missing test annotations

lboynton opened the pull request:
These tests weren't being run!

GrahamCampbell commented:
Please could you send this to 19.9. :)

GrahamCampbell commented on tests/Api/ProjectsTest.php:
```diff
@@ -1957,6 +1960,9 @@ public function shoudGetApprovalsConfiguration(): void
+    /**
+     * @test
+     */
     public function shoudGetApprovalRules(): void
```
`shoudGetApprovalsRules`

lboynton replied:
Done :)

lboynton committed:
Add missing test annotations

These tests weren't being run! Also fixed failing test due to incorrect method name being called.

lboynton commented:
> Please could you send this to 19.9. :)

Rebased on 9.19, hope that's correct :)

GrahamCampbell commented:
Yep, this looks great. :+1:

glensc commented:
I find it more convenient to use matching pattern for method names: `testSomething`, so they get picked up automatically. less clutter as well.
